python/tidy/servo_tidy/tidy.py

```diff
@@ -19,6 +19,22 @@
 import sys
 from licenseck import MPL, APACHE, COPYRIGHT, licenses_toml, licenses_dep_toml
 import colorama
+import toml
+
+CONFIG_FILE_PATH = os.path.join(".", "servo-tidy.toml")
+
+# Default configs
+config = {
+    "skip-check-length": False,
+    "skip-check-licenses": False,
+    "ignore": {
+        # Ignore hidden files by default
+        "files": ["./.", CONFIG_FILE_PATH],
```

Comment:
In this case, let's have something like,

``` python
"files": [
    "./.",
    CONFIG_FILE_PATH,
],
```

... as a style nit, and to avoid confusion :)